lib/plugins/aws/package/compile/events/cognitoUserPool.js

```diff
@@ -171,6 +172,7 @@ class AwsCompileCognitoUserPoolEvents {
             }
 
             let customCognitoUserPoolResource;
+            const forceDeployProperty = forceDeploy ? Math.random() * 10 : 0;
```

Comment:
Let's maybe set it to `Date.now()`, and do not set `ForceDeploy` (we can assign `undefined`) if no `forceDeploy` was set

Reply:
Updated!